src/connectedk8s/.gitignore

```diff
@@ -0,0 +1,2 @@
+# Ignore temporary test output files.
+**/azext_connectedk8s/tests/latest/temp/
```

Comment:
Please add it to the end of the file instead of the beginning

Reply:
Could you output temporary files to `/temp/` instead of specific folder `/azext_connectedk8s/tests/latest/temp/`? This way, other modules with similar requirements can also reuse it